src/generate.jl

```diff
@@ -57,15 +57,15 @@ emit!(rng, g::TableGenerator, deps, state::Nothing) = emit!(rng, g, deps)
 
 
 """
-    generate([rng::AbstractRNG=GLOBAL_RNG], dag; size::Integer=10) -> Channel{Any}
+    generate([rng::AbstractRNG=GLOBAL_RNG], dag; size::Integer=10) -> Channel{Pair{Symbol, <:Any}}
 
 Traverses the `dag` and generates the records specified by the [`TableGenerator`](@ref)
 of each node. Returns a `Channel` of size `size` comprising `table_key => record` pairs.
 """
 generate(dag; kwargs...) = generate(GLOBAL_RNG, dag; kwargs...)
 
 function generate(rng::AbstractRNG, dag; size::Integer=10)
-    channel = Channel(size) do ch
+    channel = Channel{Pair{<:Symbol,<:NamedTuple}}(size) do ch
```

Comment:
should it be this?
```suggestion
    channel = Channel{Pair{<:Symbol,<:Any}}(size) do ch
```

Reply:
```suggestion
    channel = Channel{Pair{Symbol,<:Any}}(size) do ch
```

woops yeah missed that